plugin/evm/service.go

```diff
@@ -43,3 +44,11 @@ func (api *SnowmanAPI) IssueBlock(ctx context.Context) error {
 	api.vm.builder.signalTxsReady()
 	return nil
 }
+
```

Comment:
let's write this in a separate file? maybe in `plugin/evm/orderbook/service.go` (as per our refactoring plan)

Reply:
refactored code. Please recheck